meson.build

```diff
@@ -2,7 +2,7 @@
 project(
     'pandas',
     'c', 'cpp', 'cython',
-    version: run_command(['generate_version.py', '--print'], check: true).stdout().strip(),
+    version: '2.3.0',
```

Comment:
@lithomas1 so I'm just trying to upload the 3.9 packages with twine from this PR, and originally I'm getting

```python
The use of local versions in <Version('2.3.0+3.g313260cf75')> is not allowed
```

So I'm guessing this line is adding the extra git hash in the metadata. Do you know if I'll need to hard code the version elsewhere?

Reply:
We'll take it :).